client/src/components/Quiz.js

```diff
@@ -17,8 +17,11 @@ const Quiz = () => {
     setUserAnswers({});
     setCurrentQuestionIndex(0);
     setShouldDisplayScore(false);
-    const quizData = retrieveQuizData(quizName);
-    setQuestions(quizData);
+    getData(quizName)
+      .then(res => {
+        setQuestions(res);
+      })
+      .catch(err => console.log('Error', err));
```

Comment:
Generally we don't do console logs in the frontend, so we can remove this. We generally either:

- Swallow errors, i.e. `.catch(() => {})`, or
- Display an error message to the user

Up to you which option you choose!

Reply:
I will fix that.

Reply:
At this moment I will leave it as option 1 - Swallow error, as we have another message in case there are no questions to display.